File: Packages/Syntax/Tests/SyntaxTests/Fixtures/sample.go
package main

import (
	"fmt"
)

type User struct {
	ID   int
	Name string
}

func greet(user User) string {
	return fmt.Sprintf("Hello, %s", user.Name)
}

func main() {
	users := []User{{ID: 1, Name: "Devys"}}
	for _, user := range users {
		fmt.Println(greet(user))
	}
}

File: Packages/Syntax/Tests/SyntaxTests/Fixtures/sample.cs
using System;
using System.Collections.Generic;

public record User(int Id, string Name);

public static class Program {
    public static void Main() {
        var users = new List<User> { new User(1, "Devys") };
        foreach (var user in users) {
            Console.WriteLine($"Hello {user.Name}");
        }
    }
}
